doc: Add sample common issues. (#88)
googleapis/java-pubsublite-spark: samples/README.md

3. Delete Dataproc cluster.
   ```sh
   gcloud dataproc clusters delete $CLUSTER_NAME --region=$REGION
   ```

## Common issues
1. Permission not granted. <br>

>    This could happen when creating topic and subscription, or submitting a job to your Dataproc cluster.

Reviewer: ```suggestion
   This could happen when creating a topic and a subscription, or submitting a job to your Dataproc cluster.
```

>    This could happen in any step, for example when creating topic and subscription, or submitting job to Dataproc cluster.

Reviewer: ```suggestion
   This could happen in when creating topic and subscription, or submitting job to Dataproc cluster.
```

>    Make sure the account/service account is granted with edit permissions for Pub/Sub Lite and Dataproc. Also the 

Reviewer: ```suggestion
   Make sure the account/service account has `editor` permission for Pub/Sub Lite and Dataproc. 
```

>    Dataproc cluster should have `scope=cloud-platform` to be able to access other cloud resources within the same

Reviewer: ```suggestion
   Your Dataproc cluster also needs `scope=cloud-platform` to access other services and resources within the same
```

>    project such as Pub/Sub Lite.

Reviewer: ```suggestion
   project.
   You should also make sure that your `gcloud` and your `GOOGLE_APPLICATION_CREDENTIALS`  access the same project. You can see which project your `gcloud` and `gstuil` commands use with `gcloud config get-value project`.
```

> 2. Dataproc job shows ClassNotFound or similar exceptions. <br>

Reviewer: ```suggestion
2. Dataproc job shows `ClassNotFound` or similar exceptions. <br>
```

>    Make sure the Dataproc cluster you created has images with [supported spark versions](https://github.com/googleapis/java-pubsublite-spark#compatibility). 

Reviewer: ```suggestion
   Make sure the Dataproc cluster you created uses images of [supported spark versions](https://github.com/googleapis/java-pubsublite-spark#compatibility). 
```

3. Delete Dataproc cluster.
   ```sh
   gcloud dataproc clusters delete $CLUSTER_NAME --region=$REGION
   ```

## Common issues
1. Permission not granted. <br>
   This could happen when creating topic and subscription, or submitting job to Dataproc cluster.
   Make sure the account/service account has `editor` permission for Pub/Sub Lite and Dataproc. 
   Your Dataproc cluster also needs `scope=cloud-platform` to access other services and resources within the same
   project.

>    You should also make sure that your `gcloud` and your `GOOGLE_APPLICATION_CREDENTIALS`  access the same project. You can see which project your `gcloud` and `gstuil` commands use with `gcloud config get-value project`.

Reviewer: ```suggestion
   Your `gcloud` and `GOOGLE_APPLICATION_CREDENTIALS`  should access the same project. Check out which project your `gcloud` and `gstuil` commands use with `gcloud config get-value project`.
```

> 2. Dataproc job shows `ClassNotFound` or similar exceptions. <br>

Reviewer: ```suggestion
2. Your Dataproc job fails with `ClassNotFound` or similar exceptions. <br>
```
